modules/income_and_assets/spec/lib/income_and_assets/pdf_fill/forms/va21p0969_spec.rb

```diff
@@ -17,8 +17,6 @@ def basic_class
   describe '#merge_fields' do
     it 'merges the right fields', run_at: '2016-12-31 00:00:00 EDT' do
```

Comment:
@ericboehs this spec tests that the example `form_data` in `vets_json_schema` transforms correctly into the format expected by the `pdf_fill` lib which is captured in the `merge_field` JSON file that is also updated in this PR

Reply:
By removing the line: `form_data.delete('royaltiesAndOtherProperties')` here we are enabling the `royaltiesAndOtherProperties` section of the form_data to be tested against the expected output in `merge_fields.json`